Dockerfile

```diff
@@ -1,14 +1,19 @@
 FROM ruby:2.6.3
 
-ENV LANG="en_US.UTF-8" \
+ENV LANG="C.UTF-8" \
     APP_HOME="/app" \
     BUNDLE_SILENCE_ROOT_WARNING="1"
 
 RUN mkdir -p $APP_HOME
 
 WORKDIR $APP_HOME
 
+COPY Gemfile Gemfile.lock $APP_HOME/
+
 RUN bundle config --global git.allow_insecure true && \
-    bundle config github.https true
+    bundle config github.https true && \
+    bundle install
```

Comment:
I guess we need to change directory at some point to be able to `bundle install`